manifests/init.pp

```diff
@@ -356,6 +359,7 @@
   Array[String]                                   $jvm_options,
   Optional[Variant[String, Hash]]                 $license,
   Stdlib::Absolutepath                            $logdir,
+  String                                          $logdir_mode,
```

Comment:
Stdlib has a Filemode type we can use. Also prefer setting the default value here so that puppet-strings can get its value:
```suggestion
  Stdlib::Filemode                                $logdir_mode = '2750',
```